docs/installing.rst

```diff
@@ -116,10 +124,10 @@ If you want to have access to the sources you can run the installation in a slig
 
   git clone https://github.com/ICRAR/daliuge
 
-Then perform the following steps to setup and install |daliuge| into an isolated environment::
+Perform the following steps to setup and install |daliuge| into a virtual environment::
 
   cd daliuge
-  make virtualenv
+  python -m venv .venv
```

Comment:
Is there a reason you have modified this? `make virtualenv` performs this exact instruction.

Reply:
Mostly just to bring attention to this line to ensure it'scompatible/consistent with the `pip` install environment recommendation

Reply:
Just some extra context, I try to avoid tooling dictating how virtual environments are setup, mainly for when local development involves switching between python versions